packages/angular-sdk-components/README.md

```diff
@@ -1,24 +1,3 @@
-# AngularSdkComponents
+# @pega/angular-sdk-components
 
-This library was generated with [Angular CLI](https://github.com/angular/angular-cli) version 16.2.0.
-
-## Code scaffolding
-
-Run `ng generate component component-name --project angular-sdk-components` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module --project angular-sdk-components`.
-> Note: Don't forget to add `--project angular-sdk-components` or else it will be added to the default project in your `angular.json` file.
-
-## Build
-
-Run `ng build angular-sdk-components` to build the project. The build artifacts will be stored in the `dist/` directory.
-
-## Publishing
-
-After building your library with `ng build angular-sdk-components`, go to the dist folder `cd dist/angular-sdk-components` and run `npm publish`.
-
-## Running unit tests
-
-Run `ng test angular-sdk-components` to execute the unit tests via [Karma](https://karma-runner.github.io).
-
-## Further help
-
-To get more help on the Angular CLI use `ng help` or go check out the [Angular CLI Overview and Command Reference](https://angular.io/cli) page.
+This Node package comprises standalone Angular components that are designed to be utilized by the Angular SDK.
```

Comment:
We can put the link of Angular SDK repo here probably since `Angular SDK` might confuse people?